projects/core/src/user/store/effects/user-replenishment-orders.effect.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
+import { Actions } from '@ngrx/effects';
+import { provideMockActions } from '@ngrx/effects/testing';
+import { cold, hot } from 'jasmine-marbles';
+import { Observable, of, throwError } from 'rxjs';
+import { ReplenishmentOrderList } from '../../../model/replenishment-order.model';
+import { UserReplenishmentOrderAdapter } from '../../connectors/replenishment-order/user-replenishment-order.adapter';
+import { UserReplenishmentOrderConnector } from '../../connectors/replenishment-order/user-replenishment-order.connector';
+import { UserActions } from '../actions/index';
+import * as fromEffect from './user-replenishment-orders.effect';
+
+const mockUserReplenishmentOrders: ReplenishmentOrderList = {
+  replenishmentOrders: [],
+  pagination: {},
+  sorts: [],
+};
+
+describe('User Replenishment Orders effect', () => {
+  let userReplenishmentOrdersEffect: fromEffect.UserReplenishmentOrdersEffect;
+  let replenishmentOrderConnector: UserReplenishmentOrderConnector;
+  let actions$: Observable<Actions>;
```

Comment:
when it was any, and I said to add the type, the type is `Action` and not Actions